app/serializers/basic_category_serializer.rb

```diff
@@ -31,6 +31,7 @@ class BasicCategorySerializer < ApplicationSerializer
              :default_list_filter,
              :minimum_required_tags,
              :navigate_to_first_post_after_read,
+             :default_slow_mode_seconds,
```

Comment:
Let's ship this column as part of [CategorySerializer](https://github.com/discourse/discourse/blob/f26acb4b634e148bf0c1f1dfc158859329e4beae/app/serializers/category_serializer.rb) only since the column is not useful in a list.